modules/billing/variables.tf

```diff
@@ -0,0 +1,133 @@
+  #   subaccounts = optional(set(string))
+  #   # (Optional) A single label and value pair specifying that usage from only this set of labeled resources should be included in the budget.
+  #   labels = optional(map(string))
+  # })
+  description = "(Optional) Filters that define which resources are used to compute the actual spend against the budget."
+  default     = null
+}
+
+variable "notifications" {
+  type = any
+  # type = object({
+  #   # (Optional) The name of the Cloud Pub/Sub topic where budget related messages will be published, in the form `projects/{project_id}/topics/{topic_id}`. Updates are sent at regular intervals to the topic.
+  #   pubsub_topic = optional(string)
+  #   #(Optional) The schema version of the notification. It represents the JSON schema as defined in https://cloud.google.com/billing/docs/how-to/budgets#notification_format.
+  #   schema_version = optional(string)
+  #   # (Optional) The full resource name of a monitoring notification channel in the form `projects/{project_id}/notificationChannels/{channel_id}`. A maximum of 5 channels are allowed.
+  #   monitoring_notification_channels = optional(set(string))
+  #   # (Optional) Boolean. When set to true, disables default notifications sent when a threshold is exceeded. Default recipients are those with Billing Account Administrators and Billing Account Users IAM roles for the target account.
+  #   disable_default_iam_recipients = optional(bool)
+  # })
+  description = "(Optional) Defines notifications that are sent on every update to the billing account's spend, regardless of the thresholds defined using threshold rules."
+  default     = null
+}
+
+
+# ----------------------------------------------------------------------------------------------------------------------
+# MODULE CONFIGURATION PARAMETERS
+# These variables are used to configure the module.
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "module_enabled" {
+  type        = bool
+  description = "(Optional) Whether to create resources within the module or not."
+  default     = true
+}
+
+variable "module_timeouts" {
+  description = "(Optional) How long certain operations (per resource type) are allowed to take before being considered to have failed."
+  type        = any
+  # type = object({
+  #   google_billing_budget = optional(object({
+  #     create = optional(string)
+  #     update = optional(string)
+  #     delete = optional(string)
+  #   }))
+  # })
+  default = {}
+}
+
+variable "module_depends_on" {
+  type        = any
+  description = "(Optional) A list of external resources the module depends on."
+  default     = []
+}
+
+variable "project_id" {
+  description = "project ID"
+  type = string
+}
+
+
+variable "owner" {
+  description = "Owner of the project"
+  type = string
+}
+
+variable "manager" {
```

Comment:
This variable should probably be moved up outside the `MODULE CONFIGURATION PARAMETERS` section